### DISCUSSION TITLE
BleachBit removes folders that shouldn't touch?

### DISCUSSION THREAD

**Discussion by PavanLuca** [bleachbit/bleachbit#1646] [Q&A]
I just discovered that Flyre lost its icon on the desktop shortcut because the icon path isn't where it should be.
![Schermata da 2024-03-13 16-36-46](https://github.com/bleachbit/bleachbit/assets/123581658/f5f7abe4-21d4-4bfd-b25a-898da9960966)
Does BleachBit delete certain folders of software paths? Of course I never deleted that folder, I had no reason for doing it.

**Comment by az0** [comment]
BleachBit has an option "broken desktop shortcuts," which deletes shortcuts that are broken like a shortcut for an application that is not installed. BleachBit 4.9.1 has fixes for this feature.

If you see it detecting .desktop files that are valid, submit a sample of the .desktop file as an issue ticket. (Don't report bugs as discussions, as discussions don't get triaged.)

**Reply by PavanLuca** [reply]
Sorry, it's not needed anymore, now I know what could have happened (now I don't have that game), after updating some games their icons disappear because their folder where the icon is is removed. For example after updating Interstellar Armada: Galactic Ace the icon disappeared but when I checked its folders I saw that where the icon should be the folder wasn't there.
![image](https://github.com/user-attachments/assets/e174c65d-4039-4eb5-8d8c-960a87aad752)
